esp/bindings/http/platform/httptransport.cpp

```diff
@@ -1112,6 +1181,14 @@ bool CHttpMessage::isFormSubmission()
         hasContentType(HTTP_TYPE_FORM_ENCODED));
 }
 
+void CHttpMessage::enableCompression()
+{
+    const char* off = getenv("ESP_COMPRESSION_OFF");
```

Comment:
Remind me where all getenv values can be set?  Seem to remember differences in ESP, roxie, and thor.

Reply:
The value can be passed in from command line shell. This is put in as a last-resort runtime fix for just in case, I don't expect it to ever be used (or hope that it will never be used).